roxie/ccd/ccdlistener.cpp

```diff
@@ -1308,6 +1302,7 @@ class RoxieWorkUnitWorker : public RoxieQueryWorker
             StringBuffer s;
             logctx.getStats(s);
 
+            //MORE: logctx.queryActiveSpan()->getLogPrefix() or similar.
```

Comment:
Agreed we need a consistent way of doing this for each flavor of tracing.  And longer term we will need to make sure there is enough info to tie traces and log lines together so the tools can hop between the two.